packages/datatrak-web/src/views/LandingPage/RecentSurveysSection.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { SectionHeading } from './SectionHeading';
+
+const RecentSurveys = styled.section`
+  grid-area: recentSurveys;
+  display: flex;
+  flex-direction: column;
+  height: 8rem;
+`;
+
+const ScrollBody = styled.div`
+  background: white;
```

Comment:
Another one where we could use the theme variables. I also wonder if we make a reusable component for these, since there seem to be a few similar components like this?